packages/manager/apps/dedicated/client/app/billing/confirmTerminate/Service.class.js

```diff
@@ -0,0 +1,13 @@
+export default class Service {
+  constructor(resource) {
```

Comment:
Wasn't the correct pattern 
```suggestion
  constructor({ property1, property2 }) {
```
?

I don't remember why we had to do this

Reply:
I think I used this shortcut because I wasn't sure the property I was going to need. It might be better with your proposal as it is explicit

Reply:
Yes it is better to have explicit properties. 
We got some trouble in past with remote paging in the datagrid from `ovh-ui-angular` if `resource` contains a `$promise` property (cf: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-datagrid/src/paging/datagrid-paging-abstract.js#L88)